packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -113,4 +148,81 @@ export class QuoteRequestor {
 
         return orders;
     }
+
+    public async requestRfqtIndicativeQuotesAsync(
```

Comment:
Seems like a lot of duplicated code with `requestRfqtFirmQuotesAsync`. Any thoughts on abstracting some shared logic?

Reply:
I think, at least in this iteration if not the last (I can't remember...), these functions are pretty distinct.  They're both limited in scope to dealing with the specific endpoints they hit, with the bulk of them coming from unpacking the response types, which differ in each case.  I do still see some patterns of duplication, but IMO they're limited.  Let me know what you think in this round.

Reply:
Good point, they are more distinct then I realized on first pass -- i'm cool with leaving them as-is 👍